### DISCUSSION TITLE
Android target sdk to 33

### DISCUSSION THREAD

**Discussion by ghost** [godotengine/godot-proposals#4683] [Platforms]
On godot we can't target sdk 33. We can only target 30. As target sdk 33 will make our game supportive on more device. I think the engine should give us the ability to target sdk 33.

**Comment by filipworksdev** [comment]
I think is recommended to use target 31 starting with 2022 https://developer.android.com/google/play/requirements/target-sdk

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot/issues/48636. No need to open feature proposals for this, as platform support updates aren't considered new features so to speak.
